postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
why do we need this config ?

Reply:
Since 6.2.x no longer included PostCSS in the core library, `@storybook/addon-postcss` had to be manually added as a dependency and for this to work, a `postcss.config.js` file had to be added as well. (See: [PR](https://github.com/storybookjs/storybook/pull/13669))